src/device_handle.rs

```diff
@@ -109,18 +110,18 @@ impl<'a> Iterator for ClaimedInterfacesIter<'a> {
 }
 
 /// A handle to an open USB device.
-#[derive(Eq, PartialEq)]
```

Comment:
If i remember correctly it still possible to use `#[derive(Eq)]` here. As you manual implement `PartialEq`

Reply:
I tried, but the compiler complains.

        Checking rusb v0.9.3 (/data/surban/dev/rusb)
    error[E0277]: the trait bound `Mutex<ClaimedInterfaces>: Eq` is not satisfied
       --> src/device_handle.rs:117:5
        |
    113 | #[derive(Eq)]
        |          -- in this derive macro expansion
    ...
    117 |     interfaces: Mutex<ClaimedInterfaces>,
        |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Eq` is not implemented for `Mutex<ClaimedInterfaces>`
        |
    note: required by a bound in `AssertParamIsEq`
       --> /home/surban/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:313:31
        |
    313 | pub struct AssertParamIsEq<T: Eq + ?Sized> {
        |                               ^^ required by this bound in `AssertParamIsEq`
        = note: this error originates in the derive macro `Eq` (in Nightly builds, run with -Z macro-backtrace for more info)
    
    For more information about this error, try `rustc --explain E0277`.
    error: could not compile `rusb` (lib) due to previous error